cincinnati/src/common.rs

```diff
@@ -0,0 +1,17 @@
+extern crate regex;
+use self::regex::Regex;
+use failure::Error;
+
+/// Strip any leading and trailing slashes
+pub fn parse_path_namespace(path_namespace: &str) -> Result<String, Error> {
+    if path_namespace == "" {
+        return Ok(path_namespace.to_string());
+    };
+
+    let stripped_leading = Regex::new(r"^/+")?.replace(path_namespace, "").to_string();
```

Comment:
Just out of curiosity, why not https://doc.rust-lang.org/std/string/struct.String.html#method.trim_start_matches?

Reply:
Simply because I missed it while reading through the docs :-) even better there's one which works for prefixes and suffixes in the same call!